blocks/property-search-bar/common-function.js

```diff
@@ -0,0 +1,254 @@
+    }
+  }
+
+  return output;
+}
+
+export function addOptions(filterName, defaultValue = '', mode = '', name = '') {
+  const selectedHtml = mode === 'multi'
+    ? `<div class="select-selected" role="button" aria-haspopup="listbox" name=${name}>${defaultValue}</div>
+            `
+    : '';
+
+  return `<section>
+        <div>
+            <select class="hide" aria-label="${defaultValue}">${buildSelectOptions(filterName, defaultValue, mode)}</select>
+            ${selectedHtml}
+            <ul class="select-item" role="listbox">${buildListBoxOptions(filterName, defaultValue, mode)}</ul>
+        </div>
+    </section>`;
+}
+
+export function formatInput(string) {
+  return string.replace(/[/\s]/g, '-').toLowerCase();
+}
+
+export function getPlaceholder(country) {
+  return country === 'US' ? 'Enter City, Address, Zip/Postal Code, Neighborhood, School or MLS#' : 'Enter City';
+}
+
+export function addRangeOption(filterName) {
+  const config = { ...TOP_LEVEL_FILTERS, ...EXTRA_FILTERS };
+  const { label } = config[filterName];
+  const filterLabel = label.charAt(0).toLocaleUpperCase() + label.slice(1).toLowerCase();
+  const fromLabel = 'No Min';
+  const toLabel = 'No Max';
+  const maxLength = 14;
+  let output = '';
+  if (filterName === 'Price') {
+    output = `<div class="multiple-inputs">
+                <div id="Min${filterLabel}" class="input-dropdown">
+                <input type="text" maxLength="${maxLength}" list="listMin${filterLabel}"
+                    name="Min${filterLabel}" aria-describedby="Min${filterLabel}"
+                    placeholder="${fromLabel}" aria-label="Minimum ${filterLabel}"
+                    class="price-range-input min-price">
+                <datalist id="listMinPrice" class="list${filterLabel}"></datalist>
+            </div>
+        <span class="range-label text-up">to</span>
+        <div id="Max${filterLabel}" class="input-dropdown">
+            <input type="text" maxLength="${maxLength}" list="listMax${filterLabel}" name="Max${filterLabel}" aria-describedby="Max${filterLabel}"
+                  placeholder="${toLabel}" aria-label="Maximum ${filterLabel}"
+                  class="price-range-input max-price">
+            <datalist id="listMax${filterLabel}" class="list${filterLabel}"></datalist>
+            </div>
+        </div>`;
+  }
+  if (filterName === 'LivingArea' || filterName === 'YearBuilt') {
+    const fromName = filterName === 'LivingArea' ? 'MinLivingArea' : '';
+    const toName = filterName === 'LivingArea' ? 'MaxLivingArea' : '';
+    output = `<div class="multiple-inputs">
+                ${addOptions(filterName, fromLabel, 'multi', fromName)}
+                <span class="range-label text-up">to</span>
+                ${addOptions(filterName, toLabel, 'multi', toName)}
+                </section>
+            </div>
+            `;
+  }
+  return output;
+}
+
+function abbrNum(d, h) {
```

Comment:
I'm not really sure what this is doing: meaningful variable names and some comments would go a long way to remedying that.